### ISSUE TITLE
Training with labeled data

### ISSUE THREAD

**Issue by doantientai** [taki0112/BigGAN-Tensorflow#13]
Hi @taki0112 , thanks for the amazing work! I have a small question.

In the official source code, it requires training data to be organized in directories according to their labels (https://github.com/ajbrock/BigGAN-PyTorch/blob/master/datasets.py#L82)

But in your implementation, all images can be put in the same folder (as described in Readme) So I wonder if this version of Biggan functions the same way as the original one?
